### PR TITLE
Started adding options. Not currently working.

### PR DESCRIPTION
**Pull request by prescient-slime**
Can't get Cobra to respect flags that I'm adding to the root command. I'll figure it out tomorrow. I like Cobra's structure I just feel like I'm doing something stupid.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,15 +1,64 @@
...
+		Use: "put usage example here",
+		//TraverseChildren: true,
+		Short: "This program shows disk usage",
+		Long:  "Put longer version of Short here",
+		Run: func(cmd *cobra.Command, args []string) {
+			v, _ := cmd.Flags().GetBool("versFlag")
+			if v {
+				version()
+			} else {
+				fmt.Println("it brokey")
```
Comment (davleop) [inline main.go:33]: If something breaks the program, we should probably use `log.Fatal` or something in the `log` lib

**Comment by davleop** [comment]
I'll probably change the base branch to merge into to be the `development` branch rather than `main`. So `main` will be our mvp

File: main.go [hunk 2]
```diff
@@ -1,15 +1,107 @@
...
+	flags.BoolVarP(&versionFlag, "version", "v", false, "-v shows the current version of godu")
+
+}
+
 func main() {
-    p := tea.NewProgram(InitialModel(), tea.WithAltScreen())
-    if err := p.Start(); err != nil {
-        log.Fatal(err)
-    }
+	//commented out to avoid going into tui every time I tested code.
```
Comment (davleop) [inline main.go:96]: We can probably delete the comments for this merge. I'll have something new to replace the Bubble Tea stuff in the next PR I make

**Review by davleop — CHANGES_REQUESTED** [review]
See comment made for main.go.

**Comment by davleop** [comment]
#3 Got merged. Do we need to keep this PR?

**Comment by prescient-slime** [comment]
> #3 Got merged. Do we need to keep this PR?

No we don't. It's fine to close.

**Pull request closed without merging**